caravel/assets/stylesheets/less/bootswatch.less

```diff
@@ -415,26 +414,29 @@ input[type="checkbox"],
 
 .nav-tabs {
   > li > a,
+  > li > a:hover
   > li > a:focus {
     margin-right: 0;
     background-color: transparent;
-    border: none;
+    border: 1px solid #ccc;
+    margin-right: 1px;
+    border-radius: 10px 10px 0px 0px;
```

Comment:
Might be a bit of a surgery to get this in, here's what I have now:
<img width="308" alt="screen shot 2016-08-10 at 9 13 18 pm" src="https://cloud.githubusercontent.com/assets/487433/17578566/51632edc-5f3f-11e6-8136-cf1fc91695f7.png">